arch/arm64/configs/sc7280_defconfig

```diff
@@ -428,6 +426,7 @@ CONFIG_QCOM_CLK_RPMH=y
 CONFIG_SC_CAMCC_7280=y
 CONFIG_SC_DISPCC_7280=y
 CONFIG_SC_GPUCC_7280=y
+CONFIG_SC_LPASSCC_7280=y
```

Comment:
I don't think we're going to use that one, this driver touches registers that are owned by adsp on our devices, only on Chromebooks this is useful